modin/pandas/test/dataframe/test_protocol.py

```diff
@@ -0,0 +1,244 @@
+import modin.pandas as pd
```

Comment:
please also add a [license header](https://github.com/modin-project/modin/blob/edb181b34f42d34a4c5ded04a14033f7c7459c13/modin/__init__.py#L1-L12), every file should have it

Reply:
@dchigarev I added this